code/modules/antagonists/heretic/items/lintel.dm

```diff
@@ -64,8 +74,10 @@
 	new /obj/effect/temp_visual/paper_scatter(turf_target)
 	playsound(turf_target, 'sound/magic/smoke.ogg', 30)
 	new barrier_type(turf_target, user)
-	uses--
-	if(uses <= 0)
-		to_chat(user, span_warning("[src] falls apart, turning into ash and dust!"))
-		qdel(src)
+	charges--
+	charge_timers.Add(addtimer(CALLBACK(src, PROC_REF(recharge)), charge_time, TIMER_STOPPABLE))
```

Comment:
why does `charge_timers` exist if you never do anything with it besides *add* timers to it?